ProcessMaker/Repositories/CaseRepository.php

```diff
@@ -69,34 +68,27 @@ public function create(ExecutionInstanceInterface $instance): void
      */
     public function update(ExecutionInstanceInterface $instance, TokenInterface $token): void
     {
+        if (is_null($instance->case_number)) {
+            \Log::error('Case number is required. instance: ' . $instance->getKey());
+
+            return;
+        }
+
         try {
-            $case = CaseStarted::where('case_number', $instance->case_number)->first();
+            if (!$this->checkIfCaseStartedExist($instance->case_number)) {
+                \Log::error('Case number not found. instance: ' . $instance->id);
```

Comment:
How this scenario could happen, could you add some steps or a test to replicate it?

Reply:
It could happen if an exception has been thrown during the `case_started` creation, a test will be added